src/Npgsql/NpgsqlBuffer.cs

```diff
@@ -618,5 +618,71 @@ struct BitConverterUnion
         }
 
         #endregion
+
+        #region Postgis
+        //  Postgis returns bytes in ndr representation.
+        internal int PostgisReadInt32()
```

Comment:
I tought that the canonical default was little endian but I was wrong : 
http://marc.info/?l=postgis-users&m=143701246423054&w=2